packages/devextreme/js/ui/tab_panel.js

```diff
@@ -252,6 +273,51 @@ const TabPanel = MultiView.inherit({
         this._focusTarget().attr('tabIndex', -1);
     },
 
+    _getTabsOrientation() {
+        const { tabsPosition } = this.option();
+
+        if([TABS_POSITION.top, TABS_POSITION.bottom].includes(tabsPosition)) {
+            return TABS_ORIENTATION.horizontal;
+        }
+
+        if([TABS_POSITION.right, TABS_POSITION.left].includes(tabsPosition)) {
```

Comment:
I think we may simplify this and return `TABS_ORIENTATION.vertical` immediately

Reply:
We shouldn't use TABS_ORIENTATION here.

Reply:
[feccc1664117de61365ba2df824d600fa6ed06f6]